src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -107,11 +107,10 @@ public void onLoad() {
             });
         }
         catch (Exception exception) {
-            if (exception instanceof OkaeriException) {

```

Comment:
I don't think we should extract causes from specific exceptions (e.g. InvocationTargetException works similar to OkaeriException and we don't really care). We should support printing of the whole exception no matter what.

Reply:
😬 https://github.com/FunnyGuilds/FunnyGuilds/commit/9cc01857b8aba648cedd488c1e270415f8545ade